pkgs/development/lisp-modules/quicklisp-to-nix-output/clfswm.nix

```
@@ -0,0 +1,25 @@
+args @ { fetchurl, ... }:
+rec {
+  baseName = ''clfswm'';
+  version = ''20161204-git'';
+
+  description = ''CLFSWM: Fullscreen Window Manager'';
+
+  deps = [ args."clx" ];
+
+  src = fetchurl {
+    url = ''http://beta.quicklisp.org/archive/clfswm/2016-12-04/clfswm-20161204-git.tgz'';
+    sha256 = ''1jgz127721dgcv3qm1knc335gy04vzh9gl0hshp256rxi82cpp73'';
```

```suggestion
    url = "http://beta.quicklisp.org/archive/clfswm/2016-12-04/clfswm-20161204-git.tgz";
    sha256 = "1jgz127721dgcv3qm1knc335gy04vzh9gl0hshp256rxi82cpp73";
```

Can we reuse version here?